repository-management.md

```diff
@@ -0,0 +1,142 @@
+* `state:unsupported` - The issue is outside of the scope of a bug report and cannot be supported on GitHub. The reporter should be pointed towards one of the [support channels](http://wp-cli.org/#support).
+
+#### Command
+
+The labels that define what specific command a given issue/pull request is related to are prefix with `command:`.
+
+Some examples:
+
+* `command:core` - Relates to one or more of the subcommands under the `wp core` parent command.
+* `command:cli-check-update` - Relates to the `wp cli check-update` command.
+* `command:post-meta-update` - Relates to the `wp post-meta update` command.
+ 
+### Required Labels
+
+Some labels have a special meaning and/or might be used for automated workflows down the road. These are required across all official packages.
+
+#### Bug
+
+The `bug` label denotes an issue that is a confirmed bug where the code does not produce the expected result.
+
+#### Good First Issue
+
+The `good-first-issue` label denotes an issue that is a good entry point for new contributors wanting to get their feet wet.
+
+Good first issues are small in scope and don't require extensive technical expertise or historical project knowledge to get started.
+
+## Commits
+
+No direct commits are to be done against the `master` branch of the packages. All code changes need to go through a pull request workflow. 
+
+## Pull Requests
+
+All code changes go through a pull request.
+
+Every submitted pull request needs to go through a code review, and needs to be approved by at least one of the [committers](https://github.com/orgs/wp-cli/teams/committers).
+
+Non-trivial pull requests should be preceded by a related issue that defines the problem to solve and allows for discussion of the most appropriate solution before actually writing code.
+  
+If a pull request was submitted by one of the [committers](https://github.com/orgs/wp-cli/teams/committers), the submitter should set the "Reviewers" for that pull request to `wp-cli/committers` if a general code review is needed, or to one or more specific committer profiles if the expertise of a specific person is needed/wanted.
+
+If a pull request was submitted by an external contributor, the [committers](https://github.com/orgs/wp-cli/teams/committers) should be responsive and provide quick feedback to encourage further contributions.
+
+Apart from needing to be approved, pull requests also need to have their tests in a passing state before they can be merged. When both of these conditions are true, any [committer](https://github.com/orgs/wp-cli/teams/committers) can merge the pull request by:
+1. Ensuring that all applicable labels have been set.
+2. Ensuring that the correct milestone has been set.
+3. Ensuring that the branch is deleted after the merge (if applicable).
+4. Adapting the title of the pull-request as needed. The ideal pull request title is one that we can copy and paste into release notes as is.
```

Comment:
First use of "we", maybe keep it impersonal "can be copied and pasted".

Reply:
https://github.com/wp-cli/handbook/pull/127/commits/263be425d6ff582601d24334a43c5977d5de4dbf